ServiceWebsite/ServiceWebsite.AcceptanceTests/Steps/AboutYouSteps.cs

```diff
@@ -0,0 +1,22 @@
+﻿using ServiceWebsite.AcceptanceTests.Pages.IndividualPages;
+using TechTalk.SpecFlow;
+
+namespace ServiceWebsite.AcceptanceTests.Steps
+{
+    [Binding]
+    public sealed class AboutYouSteps
```

Comment:
It looks like these classes have quite a lot of duplication, both the steps classes and the POM, we can look at a pattern to make this more abstract whilst still keeping the readability if you like

Reply:
Yes, let's have a look please.